corehq/apps/app_manager/templates/app_manager/partials/modules/case_list_multi_select.html

```diff
@@ -3,6 +3,14 @@
 
 {% if request|toggle_enabled:"USH_CASE_LIST_MULTI_SELECT" %}
 
+	<script>
+		function reconcileCheckboxes() {
```

Comment:
This logic should be part of the initialization of `autoSelectEnabled` in `screen.js`

Reply:
https://github.com/dimagi/commcare-hq/pull/32303#:~:text=autoselect%20checkbox%20logic-,116b101,-allow%20auto_select%20and